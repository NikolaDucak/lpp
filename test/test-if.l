// if statement
if( true )
	println("ifstmt");

if( true ){
	println("block");
	println("stmt");
}

if( false )
	println("ifstmt");
else 
	println("elsestmt");


if( false ){
	println("ifstmt");
} else {
	println("block");
	println("stmt");
}

