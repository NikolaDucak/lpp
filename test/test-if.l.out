ifstmt
block
stmt
elsestmt
block
stmt
